src/clib/old_tests/enkf/test_enkf_gen_data_config_parse.cpp

```diff
@@ -1,165 +0,0 @@
-    config_parser_type *config = config_alloc();
-    enkf_config_node_type *enkf_config_node = NULL;
-
-    auto item = config_add_schema_item(config, GEN_DATA_KEY, false);
-    config_schema_item_set_argc_minmax(item, 1, CONFIG_DEFAULT_ARG_MAX);
-    {
-        FILE *stream = util_fopen("config.txt", "w");
-        fputs(config_string, stream);
-        fclose(stream);
-    }
-    {
-        config_content_type *content =
-            config_parse(config, "config.txt", "--", NULL, NULL, NULL,
-                         CONFIG_UNRECOGNIZED_IGNORE, true);
-        test_assert_bool_equal(parse_valid, config_content_is_valid(content));
-        if (parse_valid) {
-            const config_content_item_type *config_item =
-                config_content_get_item(content, GEN_DATA_KEY);
-            const config_content_node_type *config_node =
-                config_content_item_iget_node(config_item, 0);
-
-            enkf_config_node =
-                enkf_config_node_alloc_GEN_DATA_from_config(config_node);
-        }
-
-        config_content_free(content);
-        config_free(config);
-    }
-    return enkf_config_node;
-}
-
-void test_parse_gen_data_result() {
-    ecl::util::TestArea ta("GEN_DATA_RESULT_parse");
-    // Parse error: missing KEY
-    {
-        enkf_config_node_type *config_node =
-            parse_alloc_GEN_DATA_result("GEN_DATA\n", false);
-        test_assert_NULL(config_node);
-    }
-
-    // Validation error: missing INPUT_FORMAT:
-    {
-        enkf_config_node_type *config_node = parse_alloc_GEN_DATA_result(
-            "GEN_DATA GEN_DATA_KEY  RESULT_FILE:Results%d  REPORT_STEPS:10 \n",
-            true);
-        test_assert_NULL(config_node);
-    }
-
-    // Validation error: Invalid INPUT_FORMAT:
-    {
-        enkf_config_node_type *config_node = parse_alloc_GEN_DATA_result(
-            "GEN_DATA GEN_DATA_KEY  RESULT_FILE:Results%d  "
-            "INPUT_FORMAT:AsCiiiiii REPORT_STEPS:10 \n",
-            true);
-        test_assert_NULL(config_node);
-    }
-
-    // Validation error: missing RESULT_FILE:
-    {
-        enkf_config_node_type *config_node = parse_alloc_GEN_DATA_result(
-            "GEN_DATA GEN_DATA_KEY  INPUT_FORMAT:ASCII   REPORT_STEPS:10 \n",
-            true);
-        test_assert_NULL(config_node);
-    }
-
-    // Validation error: Invalid RESULT_FILE:
-    {
-        enkf_config_node_type *config_node = parse_alloc_GEN_DATA_result(
-            "GEN_DATA GEN_DATA_KEY  RESULT_FILE:Results INPUT_FORMAT:ASCII   "
-            "REPORT_STEPS:10 \n",
-            true);
-        test_assert_NULL(config_node);
-    }
-
-    // Validation error: Missing REPORT_STEPS:
-    {
-        enkf_config_node_type *config_node = parse_alloc_GEN_DATA_result(
-            "GEN_DATA GEN_DATA_KEY  RESULT_FILE:Results%d INPUT_FORMAT:ASCII  "
-            "\n",
-            true);
-        test_assert_NULL(config_node);
-    }
-
-    // Validation error: Invalid REPORT_STEPS
```

Comment:
yaaa now I see where these test cases came from! so it shouldn't be much work to steal the strings from the comments : )